### DISCUSSION TITLE
Adding ML to the Watcher

### DISCUSSION THREAD

**Discussion by AnilMandliya** [Waishnav/Watcher#42] [Ideas]
3 months back I wanted to do hands on Machine Learning and so I thought of tracking my screen time and then use ML to provide insights and measures for future use. I started looking for open source repo which are not too complex and came across this. I think it has very simple interface and easily readable code with good documentation. Cut to today, I am still thinking of doing it :( 
I have still have some ideas for it. I am looking for a partner with whom I can co-create this effort or a mentor to guide me through it. 
TLDR; Looking for a partner/mentor to add Machine Learning to it.
Thanks

**Comment by Waishnav** [comment]
What's your proposal?
Which features do you have in mind?
